### DISCUSSION TITLE
+ NAME таблиц для настройки системы

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#500] [Есть идея, а что если...]
Добрый день,
Заметил для себя одно неудобство при настройке системы. Т.к. многие видимые названия таблиц одинаковые, хотя их NAME отличаются - очень сложно найти нужную таблицу при настройке системы. Кое-где это удалось исправить - во временных системных таблицах, но в других местах это невозможно. Хотел предложить для удобства добавить и NAME'ы таблицы в селекты, см. скриншот. Как считаете, это возможно?
Благодарю
![Screenshot_3](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113771444/4ae7662d-ea30-4c1b-ad73-d226486096dd)

**Comment by ttm-alexeypolunin** [comment]
Было бы хорошо
